core/frontend/src/tools/ClipViewTool.ts

```diff
@@ -29,7 +29,7 @@ import { ToolAssistance, ToolAssistanceImage, ToolAssistanceInputMethod, ToolAss
 
 // cSpell:ignore geti
 
-/** @alpha An object that can react to a view's clip being changed by tools or modify handles. */
+/** @public An object that can react to a view's clip being changed by tools or modify handles. */
 export interface ViewClipEventHandler {
   selectOnCreate(): boolean; // Add newly created clip geometry to selection set and show modify controls.
```

Comment:
Turn the `//` comments into `/** doc comments */`.

Reply:
done